File: docs/visual-basic/developing-apps/windows-forms/codesnippet/CSharp/walkthrough-displaying-data-in-a-datarepeater-control-visual-studio_5.cs
            int foundIndex;
            string searchString;
            searchString = searchTextBox.Text;
            // Search for the string in the CustomerID field.
            foundIndex = customersBindingSource.Find("CustomerID", searchString);
            if (foundIndex > -1)
            {
                dataRepeater1.CurrentItemIndex = foundIndex;
            }
            else
            {
                MessageBox.Show("Item " + searchString + " not found.");
            }

File: docs/visual-basic/developing-apps/windows-forms/codesnippet/VisualBasic/walkthrough-displaying-data-in-a-datarepeater-control-visual-studio_5.vb
        Dim foundIndex As Integer
        Dim searchString As String
        searchString = SearchTextBox.Text
        ' Search for the string in the CustomerID field.
        foundIndex = CustomersBindingSource.Find("CustomerID",
         searchString)
        If foundIndex > -1 Then
            DataRepeater1.CurrentItemIndex = foundIndex
        Else
            MsgBox("Item " & searchString & " not found.")
        End If